London | IPT-MAY-25 | Tewodros Bekere | Module-Onboarding | Wireframe

iteddy16 opened the pull request:
<!--

You must title your PR like this:

REGION | COHORT_NAME | FIRST_NAME LAST_NAME | PROJ_NAME

For example,

London | May-2025 | Carol Owen | Wireframe

Complete the task list below this message.
If your PR is rejected, check the task list.

-->

## Learners, PR Template

Self checklist

- [x] I have committed my files one by one, on purpose, and for a reason
- [x] I have titled my PR with REGION | COHORT_NAME | FIRST_NAME LAST_NAME | PROJ_NAME
- [x] I have tested my changes
- [x] My changes follow the [style guide](https://curriculum.codeyourfuture.io/guides/reviewing/style-guide/)
- [x] My changes meet the [requirements](./README.md) of this task

## Changelist

-> Added Articles to the wireframe; make a column section
-> Modified the footer and added the current time. 
-> Add a link in the read more button. 

## Questions


I am trying to align the footer text to the middle position, but it is not working. I added the CSS the correct way, and I also tried to add inline style in the footer, but it is not working. What is the problem?

jenny-alexander reviewed (changes requested):
Can you remove the change related to /Form-Controls assignment? There should only be Wireframe changes in this PR.

🤔 Regarding your question about the footer not being centered:
If I'm having issues with CSS alignment, I like to add borders to see where my problem might be.
In the footer, I added a red border. I can see that your text is actually centered within the footer element. If you expand the width of the footer across the entire page, what will happen? Try it out!
<img width="535" alt="Screenshot 2025-05-11 at 6 46 25 PM" src="https://github.com/user-attachments/assets/3a56c2dc-608e-4b4b-ba88-99de5f91b83b" />

iteddy16 commented on Wireframe/index.html:
```diff
@@ -8,26 +8,57 @@
+      <article>
+        <div class="row">
+          <div class="column">
+            <img src="placeholder.svg" alt="placeholder" />
```
done

iteddy16 commented on Wireframe/index.html:
```diff
@@ -8,26 +8,57 @@
+            <a href="articles.md">Read more</a>
+          </div>
+          <div class="column">
+            <img src="placeholder.svg" alt="placeholder" />
```
done

iteddy16 commented on Wireframe/index.html:
```diff
@@ -8,26 +8,57 @@
-        <h2>Title</h2>
+        <img
+          src="placeholder.svg"
+          alt="placeholder"
```
done

iteddy16 commented on Wireframe/index.html:
```diff
@@ -8,26 +8,57 @@
+        <img
+          src="placeholder.svg"
+          alt="placeholder"
+          style="align-content: center"
```
done

iteddy16 commented:
Thank you for your detailed review. I think I addressed all the raised
issues and pushed to the repo. Please review it again.
Best Regards,

On Mon, May 12, 2025 at 2:06 AM Jennifer Alexander ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
>
> Can you remove the change related to /Form-Controls assignment? There
> should only be Wireframe changes in this PR.
>
> 🤔 Regarding your question about the footer not being centered:
> If I'm having issues with CSS alignment, I like to add borders to see
> where my problem might be.
> In the footer, I added a red border. I can see that your text is actually
> centered within the footer element. If you expand the width of the footer
> across the entire page, what will happen? Try it out!
> Screenshot.2025-05-11.at.6.46.25.PM.png (view on web)
> <https://github.com/user-attachments/assets/3a56c2dc-608e-4b4b-ba88-99de5f91b83b>
> ------------------------------
>
> In Wireframe/index.html
> <https://github.com/CodeYourFuture/Module-Onboarding/pull/365#discussion_r2083654057>
> :
>
> >          </p>
> -        <a href="">Read more</a>
> +        <a href="articles.md">Read more</a>
> +      </article>
> +    </main>
> +    <main>
>
> In order to follow semantic HTML best practices, there should only be 1
> <main> element.
> Further reading: https://www.w3schools.com/tags/tag_main.asp
> ------------------------------
>
> In Wireframe/index.html
> <https://github.com/CodeYourFuture/Module-Onboarding/pull/365#discussion_r2083654213>
> :
>
> >          </p>
> -        <a href="">Read more</a>
> +        <a href="articles.md">Read more</a>
> +      </article>
> +    </main>
> +    <main>
> +      <article>
> +        <div class="row">
> +          <div class="column">
> +            <img src="placeholder.svg" alt="placeholder" />
>
> If a screen reader was reading over your page, do you think the alt text
> here would be informative enough about your image?
> ------------------------------
>
> In Wireframe/index.html
> <https://github.com/CodeYourFuture/Module-Onboarding/pull/365#discussion_r2083654450>
> :
>
> > +      </article>
> +    </main>
> +    <main>
> +      <article>
> +        <div class="row">
> +          <div class="column">
> +            <img src="placeholder.svg" alt="placeholder" />
> +            <h3>What is the purpose of a wireframe?</h3>
> +            <p>
> +              Its purpose is to create a schematic in the early stages of
> +              digital product design
> +            </p>
> +            <a href="articles.md">Read more</a>
> +          </div>
> +          <div class="column">
> +            <img src="placeholder.svg" alt="placeholder" />
>
> Review alt text here and make sure it's informative enough for people
> using a screen reader.
> ------------------------------
>
> In Wireframe/index.html
> <https://github.com/CodeYourFuture/Module-Onboarding/pull/365#discussion_r2083654548>
> :
>
> >      </header>
>      <main>
>        <article>
> -        <img src="placeholder.svg" alt="" />
> -        <h2>Title</h2>
> +        <img
> +          src="placeholder.svg"
> +          alt="placeholder"
>
> Review alt text here and make sure it's informative enough for people
> using a screen reader.
> ------------------------------
>
> In Wireframe/index.html
> <https://github.com/CodeYourFuture/Module-Onboarding/pull/365#discussion_r2083655057>
> :
>
> >      </header>
>      <main>
>        <article>
> -        <img src="placeholder.svg" alt="" />
> -        <h2>Title</h2>
> +        <img
> +          src="placeholder.svg"
> +          alt="placeholder"
> +          style="align-content: center"
>
> Can you add all styling to your css file? This helps with separation of
> concerns:
>
>    - keep html about what is on the page
>    - keep css about how the page content looks
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/CodeYourFuture/Module-Onboarding/pull/365#pullrequestreview-2831543659>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BRPXOO4AR4EWKTXSSNK4YH3257XYHAVCNFSM6AAAAAB4Y2LZSWVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDQMZRGU2DGNRVHE>
> .
> You are receiving this because you were assigned.Message ID:
> ***@***.***>
>

jenny-alexander reviewed (approved):
Approved!

A suggestion about your **footer**. Currently it is stuck to the bottom of the viewport (as required by the exercise). ✅ 
However, because there isn't a background-color set, the footer gets lost in the page (it's hard to see it).
Try setting the background color to something like white and making the width 100%. This will make it look better.

iteddy16 commented:
Thank you for approving my wireframe PR. I set the background Color now the
footer looks good. Please, can you also review my Form-Control PR if you
can, here is the link
London | ITP-MAY-25 | Tewodros Bekere | Module-Onboarding | Form control by
iteddy16 · Pull Request #388 · CodeYourFuture/Module-Onboarding
<https://github.com/CodeYourFuture/Module-Onboarding/pull/388>
Best Regards,

On Mon, May 12, 2025 at 11:28 PM Jennifer Alexander <
***@***.***> wrote:

> ***@***.**** approved this pull request.
>
> Approved!
>
> A suggestion about your *footer*. Currently it is stuck to the bottom of
> the viewport (as required by the exercise). ✅
> However, because there isn't a background-color set, the footer gets lost
> in the page (it's hard to see it).
> Try setting the background color to something like white and making the
> width 100%. This will make it look better.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/CodeYourFuture/Module-Onboarding/pull/365#pullrequestreview-2834762112>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BRPXOO2CFQZ2VFUTTGWS5SL26EOBDAVCNFSM6AAAAAB4Y2LZSWVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDQMZUG43DEMJRGI>
> .
> You are receiving this because you were assigned.Message ID:
> ***@***.***>
>
